x-pack/plugins/observability/public/components/shared/exploratory_view/series_builder/series_builder.tsx

```diff
@@ -129,53 +168,122 @@ export function SeriesBuilder({
       }),
       width: '20%',
       field: 'id',
-      render: (val: string) =>
-        reportType && indexPattern ? (
-          <ReportBreakdowns seriesId={seriesId} dataViewSeries={getDataViewSeries()} />
+      render: (seriesId: string, { series: { reportType }, seriesConfig }: BuilderItem) =>
+        reportType && seriesConfig ? (
+          <ReportBreakdowns seriesId={seriesId} dataViewSeries={seriesConfig} />
         ) : null,
     },
+    ...(multiSeries
+      ? [
+          {
+            name: i18n.translate('xpack.observability.expView.seriesBuilder.actions', {
+              defaultMessage: 'Actions',
+            }),
+            align: 'center' as const,
+            width: '10%',
```

Comment:
The width of all these columns no longer adds up to 100% if multi series is true.